sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -12320,4 +12320,10 @@ ALTER TABLE testreport_history ADD COLUMN dateofresult varchar(255);
 
 INSERT INTO schema_version (version_number, comment) VALUES (511, 'Add dateOfResult to TestReports #11453');
 
+-- 2023-03-27 Limit case duplicate merging comparison based on creation date and archived status #11465
```

Comment:
Code linting fails because of these changes

Reply:
The error is received due to a configuration of CICD pipeline.